website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -934,6 +934,8 @@ If the subgraph encounters an error, that query will return both the data and a
 
 ### Grafting onto Existing Subgraphs
 
+> **Note:** it is not recommended to use grafting when initially upgrading to The Graph Network. For more information, learn more [here](/cookbook/grafting/#important-note-on-grafting-when-upgrading-to-the-network).
```

Comment:
Maybe just me, but "For more information, learn more [here]" sounds a bit redundant to me. How about simply "More information [here]" or "Learn more [here]"?

Reply:
Also, super nitpicky, but in the callouts above, you didn't use bold for "Disclaimer" and "Caution" but here you used it for "Note", and also above you capitalized the word after the `:` and here you didn't.

Reply:
Agreed - fixing that!